apps/aecore/src/aec_hard_forks.erl

```diff
@@ -54,6 +55,14 @@ check_protocol_version_validity(Version, Height) ->
 protocol_effective_at_height(H) ->
     protocol_effective_at_height(H, protocols()).
 
+-spec is_fork_height(aec_blocks:height()) -> false | {true, protocol_vsn()}.
+is_fork_height(Height) ->
```

Comment:
I wonder if it might be clearer to define this (reverse map) statically, after all it will be true _very_ rarely...

Reply:
It could be done for the static forks (i.e., for mainnet and uat) at least. But on the other hand, it is still a very small map, so it is ok for now.